mujoco_warp/_src/types.py

```diff
@@ -905,6 +905,9 @@ class Model:
     mesh_polymapadr: first polygon address per vertex        (nmeshvert,)
     mesh_polymapnum: number of polygons per vertex           (nmeshvert,)
     mesh_polymap: vertex to polygon map                      (nmeshpolymap,)
+    volume_ids: Warp volume IDs for SDF sampling             (nvolume,)
+    volumes: Warp volume objects containing mesh SDF data    (nvolume,)
```

Comment:
I would remove "mesh" from the comment and remain agnostic about where the SDF comes from (to avoid confusion with actual mujoco meshes, which are always convex for collisions).